wdio.conf.js

```diff
@@ -54,25 +60,60 @@ exports.config = {
   // https://docs.saucelabs.com/reference/platforms-configurator
   //
   capabilities: [
-    ...(process.env.WEBEX_TEST_CHROME ? [{
-      browserName: 'chrome',
-      maxInstances: 5,
-      acceptInsecureCerts: true,
-    }] : []),
-    ...(process.env.WEBEX_TEST_FIREFOX ? [{
-      browserName: 'firefox',
-      maxInstances: 5,
-      acceptInsecureCerts: true,
-    }] : []),
-    ...(process.env.WEBEX_TEST_EDGE ? [{
-      browserName: 'MicrosoftEdge',
-      maxInstances: 5,
-      acceptInsecureCerts: true,
-    }]: []),
-    ...(process.env.WEBEX_TEST_SAFARI ? [{
-      browserName: 'safari',
-      maxInstances: 5,
-    }]: []),
+    ...(process.env.WEBEX_TEST_CHROME
+      ? [
+          {
+            browserName: 'chrome',
+            maxInstances: 5,
+            acceptInsecureCerts: true,
+            'goog:chromeOptions': {
+              args: ['--use-fake-device-for-media-stream', '--use-fake-ui-for-media-stream', '--disable-infobars'],
+              prefs: {
+                'profile.default_content_setting_values.notifications': 2,
+              },
+            },
+            'sauce:options': {
+              extendedDebugging: true,
```

Comment:
if capturePerformance is not needed (is it?), you can put sauce options in a separate object:
```js
const sauceOptions = {
  'sauce:options': {
    extendedDebugging: true,
  }
};


// and then use it for each browser
...(process.env.WEBEX_TEST_CHROME ? [{
  browserName: 'chrome',
  ...sauceOptions,
```